system/logs/templates/_containerd-config.tpl

```diff
@@ -7,6 +7,17 @@ transform/protocol:
         - resource.attributes["network.protocol.name"] != nil
       statements:
         - set(attributes["network.protocol.name"], ConvertCase(attributes["network.protocol.name"], "lower"))
+
+transform/ingress:
+  error_mode: ignore
+  log_statements:
```

Comment:
This one is also defined in https://github.com/sapcc/helm-charts/blob/8bc802b3c899c31ac07836459715f85c502a5322/system/logs/templates/_openstack-config.tpl#L35-L37

Would this be a problem if both the containerd and openstack would be added at the same time?

Reply:
yes, because we have a special use case for swift to put swift logs into an extra index and we don't want to rollout an openstack config to all the kubernikus clusters.